src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -100,6 +101,12 @@ public static VariantContext liftVariant(final VariantContext source, final Inte
             builder.attribute(LiftoverVcf.ORIGINAL_START, source.getStart());
         }
 
+        if (writeOriginalAlleles && !source.getAlleles().equals(builder.getAlleles())) {
```

Comment:
So you're calling equals on 2 ArrayLists, which means that for them to be equal they must be the exact same Object.  I suppose that's what is happening in the code, but you may want to use a more explicit matching routine (or at the very least add a comment to this effect).

Reply:
I dont think this is correct.  ArrayList (from AbstractList) will return true for equals() if the elements are identical, not only if they are the exact same Object:

https://docs.oracle.com/javase/7/docs/api/java/util/List.html#equals(java.lang.Object)

Reply:
Okay, thanks for the correction.